src/components/sdk/MeetingControls/AudioInputControl.tsx

```diff
@@ -22,46 +24,56 @@ interface Props extends BaseSdkProps {
   mutedIconTitle?: string;
   /** Title attribute for the icon when unmuted, it defaults to `Microphone`. */
   unmutedIconTitle?: string;
-  /** A boolean that determines whether or not to include additional sample audio input devices, such as "None", "440 Hz". Defaults to true. This will be deprecated in the next major version. */
-  appendSampleDevices?: boolean;
 }
 
 const AudioInputControl: React.FC<Props> = ({
   muteLabel = 'Mute',
   unmuteLabel = 'Unmute',
   mutedIconTitle,
   unmutedIconTitle,
-  appendSampleDevices = true,
   ...rest
 }) => {
   const meetingManager = useMeetingManager();
   const { muted, toggleMute } = useToggleLocalMute();
-  const audioInputConfig: DeviceConfig = {
-    additionalDevices: appendSampleDevices,
-  };
-  const { devices, selectedDevice } = useAudioInputs(audioInputConfig);
+  const { devices, selectedDevice } = useAudioInputs();
+  const [error, setError] = useState('');
 
   const dropdownOptions: PopOverItemProps[] = devices.map((device) => ({
     children: <span>{device.label}</span>,
     checked: isOptionActive(selectedDevice, device.deviceId),
-    onClick: (): Promise<void> =>
-      meetingManager.selectAudioInputDevice(device.deviceId),
+    onClick: () => {
+      try {
+        meetingManager.selectAudioInputDevice(device.deviceId);
+      } catch (error) {
+        setError(error.message);
+      }
+    },
   }));
 
   return (
-    <ControlBarButton
-      icon={
-        <Microphone
-          muted={muted}
-          mutedTitle={mutedIconTitle}
-          unmutedTitle={unmutedIconTitle}
-        />
-      }
-      onClick={toggleMute}
-      label={muted ? unmuteLabel : muteLabel}
-      popOver={dropdownOptions}
-      {...rest}
-    />
+    <>
+      <ControlBarButton
+        icon={
+          <Microphone
+            muted={muted}
+            mutedTitle={mutedIconTitle}
+            unmutedTitle={unmutedIconTitle}
+          />
+        }
+        onClick={toggleMute}
+        label={muted ? unmuteLabel : muteLabel}
+        popOver={dropdownOptions}
+        {...rest}
+      />
+      {error && (
```

Comment:
Good that you handle it, but, as discussed in meetings logging the error is fine for now unless we have  gone through UX to finalize this.

Reply:
If we are doing so in one component consistency of UX is also important on the rest of controls on UX for SDK connected UI components.

Reply:
Discussed offline, I will log the error message instead of displaying via `Modal` UI component.